content/working_with/console/widgets/deployments.md

```diff
@@ -50,6 +50,7 @@ The hamburger menu (![list icon]( /images/ui/icons/list-icon.png ))
 on the right of every deployment allows performing the following operations:
 
 * Install/Uninstall deployment
+* Deploy blueprint on deployment - this option is only available when the deployment is containing `csys-obj-type: environment` label
```

Comment:
In other places you are referring to environment, so maybe here you can do it as well:
```suggestion
* Deploy blueprint on environment - this option is only available when selected deployment is an environment
```
?

BTW I hope in the future https://cloudifysource.atlassian.net/browse/RD-4113 will be implemented, so all mentions of `environment` can be linked to proper page.

Reply:
I'd be fine with that direction 💪 
But the issue is that the PR has been merged before I could even consider doing it 😅 
Sorry about that